server.js

```diff
@@ -6577,6 +6577,48 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// JetBrains Plugins repository version integration
+camp.route(/^\/jetbrains\/plugin\/(v)\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var pluginId = match[2];
+  var type = match[1];
+  var format = match[3];
+  var badgeData = getBadgeData('jetbrains plugin', data);
+  var url = 'https://plugins.jetbrains.com/plugins/list?pluginId=' + pluginId;
+
+  request(url, function(err, res, buffer) {
+    if (err || res.statusCode !== 200) {
+      badgeData.text[1] = 'inaccessible';
+      return sendBadge(format, badgeData);
+    }
+
+    xml2js.parseString(buffer.toString(), function (err, data) {
+      if (err) {
+        badgeData.text[1] = 'invalid';
+        return sendBadge(format, badgeData);
+      }
+
+      try {
+        switch (type) {
+        case 'd':
+          var plugin = data["plugin-repository"].category;
+          if (!plugin) {
+            badgeData.text[1] = 'not found';
+            return sendBadge(format, badgeData);
+          }
+          var version = data['plugin-repository'].category[0]["idea-plugin"][0].version[0];
+          badgeData.text[1] = metric(downloads);
```

Comment:
There is no `downloads` variable in this function.

Reply:
Fixed